workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWorkflowListenerProperties.java

```diff
@@ -21,9 +21,19 @@
 
 @ConfigurationProperties("conductor.workflow-status-listener.archival")
 public class ArchivingWorkflowListenerProperties {
-
+    public enum ArchivalType {
+        DEFAULT,
+        S3
+    }
     private final Environment environment;
 
+    private ArchivalType workflowArchivalType = ArchivalType.DEFAULT;
+
+    private String workflowS3ArchivalDefaultBucketName = "";
+
+    private String workflowS3ArchivalBucketRegion = "us-east-1";
```

Comment:
If we are planning to push these changes to the public Conductor repository, this needs to be configurable.

Reply:
These are configurable under following properties, which we set in our conductor_build repo:
>conductor.workflow-status-listener.archival.workflowS3ArchivalBucketRegion : us-eaast-1b
>conductor.workflow-status-listener.archival.workflowS3ArchivalDefaultBucketName : TEST_BUCKET

If these properties are not passed then default values will be applied

Reply:
Let's mandate a value to be provided here. I don't see if these defaults will hold when we push this upstream.

Reply:
Okay.